android/api.md

```diff
@@ -594,6 +609,252 @@ private static String MyOIDCAuthProvider(){
+
+**Important Note:** If you are an existing customer of AWS AppSync SDK for Android, the `useClientDatabasePrefix` has a default value of `false`.  When or if you choose to use multiple `AWSAppSyncClient` objects, turning on `useClientDatabasePrefix` will change the location of the caches used by the client. The caches will not be automatically moved; you are responsible for the migration of any data within the caches that you wish to keep and deleting the old cache on the device.
+
+The following snippets highlight the new values in the `awsconfiguration.json` and the client code configurations.
+
+**awsconfiguration.json based client instantiation**
+
+The `friendly_name` illustrated here is created from a prompt from Amplify CLI. There are 4 clients in this configuration that connect to the same API except that they use different `AuthMode` and `ClientDatabasePrefix`.
+
+```
+{
+  "Version": "1.0",
+  "AppSync": {
+    "Default": {
+      "ApiUrl": "https://xyz.us-west-2.amazonaws.com/graphql",
+      "Region": "us-west-2",
+      "AuthMode": "API_KEY",
+      "ApiKey": "da2-xyz",
+      "ClientDatabasePrefix": "friendly_name_API_KEY"
+    },
+    "friendly_name_AWS_IAM": {
+      "ApiUrl": "https://xyz.us-west-2.amazonaws.com/graphql",
+      "Region": "us-west-2",
+      "AuthMode": "AWS_IAM",
+      "ClientDatabasePrefix": "friendly_name_AWS_IAM"
+    },
+    "friendly_name_AMAZON_COGNITO_USER_POOLS": {
+      "ApiUrl": "https://xyz.us-west-2.amazonaws.com/graphql",
+      "Region": "us-west-2",
+      "AuthMode": "AMAZON_COGNITO_USER_POOLS",
+      "ClientDatabasePrefix": "friendly_name_AMAZON_COGNITO_USER_POOLS"
+    },
+    "friendly_name_OPENID_CONNECT": {
+      "ApiUrl": "https://xyz.us-west-2.amazonaws.com/graphql",
+      "Region": "us-west-2",
+      "AuthMode": "OPENID_CONNECT",
+      "ClientDatabasePrefix": "friendly_name_OPENID_CONNECT"
+    }
+  }
+}
+```
+
+The `useClientDatabasePrefix` is added on the client builder which signals to the builder that the `ClientDatabasePrefix` should be used from the AWSConfiguration object (*awsconfiguration.json*).
+
+```java
+AWSAppSyncClient client = AWSAppSyncClient.builder()
+                               .context(getApplicationContext())
+                               .awsConfiguration(new AWSConfiguration(getApplicationContext()))
+                               .useClientDatabasePrefix(true)
+                               .build();
+```
+
+The following creates a client factory to retrieve the client based on the need to operate in public (API Key) or private (AWS IAM) authorization mode.
```

Comment:
"The following creates a client factory to retrieve the client based on the need to operate in public (API Key) or private (AWS IAM) authorization mode." --> 
"The following code creates a client factory to retrieve the client based on the authorization mode: public (API Key) or private (AWS IAM)."

Reply:
Changed,